vector/src/main/java/im/vector/fragments/roomwidgets/RoomWidgetPermissionBottomSheet.kt

```diff
@@ -53,8 +52,8 @@ class RoomWidgetPermissionBottomSheet : VectorBaseBottomSheetDialogFragment() {
 
     @BindView(R.id.bottom_sheet_widget_permission_owner_avatar)
     lateinit var authorAvatarView: ImageView
-
-    private val sharedActivityViewModel: RoomWidgetViewModel by activityViewModel()
+    
+    var onFinish: ((Boolean) -> Unit)? = null
```

Comment:
The sharedViewModel was better, no?

Reply:
It makes the bottom sheet more generic, can be used in other activity easier